src/rpc/blockchain.cpp

```diff
@@ -1479,6 +1479,9 @@ UniValue getchaintxstats(const JSONRPCRequest& request)
             "{\n"
             "  \"time\": xxxxx,        (numeric) The timestamp for the statistics in UNIX format.\n"
```

Comment:
Suggestion: improve this help message as part of this PR. Perhaps:

`The timestamp for the final block in the window in UNIX format.`